### PR TITLE
Created shortcut to reset layout.

### PR DESCRIPTION
**Pull request by RichardTysonIsis**
### Description of work

I added a shortcut to reset a perspective's layout. The shortcut is "Alt+Shift+O". This is a bit awkward, "O" isn't the first letter that pops to mind when thinking of "Reset Layout" but  "Alt+Shift+R", "Alt+Shift+E", "Alt+Shift+A", "Alt+Shift+L", "Alt+Shift+Y" are already taken. "Alt+Shift+Z" could maybe be better as it is close to "Ctrl+Z" that is often used to undo the last change?

**Note: it has been implemented as Alt+Shift+Z**

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/3421

### Acceptance criteria

- "Alt+Shift+O" resets the current perspective's layout.
- "Alt+Shift+O" is a suitable keybinding.

### Unit tests

None required.

### System tests

None required.

### Documentation
None found.

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher/src/uk/ac/stfc/isis/ibex/e4/ui/perspectiveswitcher/views/PerspectiveSwitcherView.java [hunk 1]
```diff
@@ -1,7 +1,6 @@
 package uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher.views;
 
 import java.util.Collections;
-import java.util.Comparator;
 import java.util.List;
```
Comment (RichardTysonIsis) [inline base/uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher/src/uk/ac/stfc/isis/ibex/e4/ui/perspectiveswitcher/views/PerspectiveSwitcherView.java]: This import wasn't used and was causing a checkstyle error.

**Comment by kjwoodsISIS** [comment]
I think "Alt-Shift+Z" is better than "Alt+Shift+O".  "Alt-Shift+Z" feels more memorable (i.e. you are more likely to remember it when you've got your display into a mess).

**Comment by RichardTysonIsis** [comment]
Ok great, I just made the change!

File: base/uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher/src/uk/ac/stfc/isis/ibex/e4/ui/perspectiveswitcher/views/PerspectiveSwitcherView.java (hunk 1 above)
Comment (Tom-Willemsen) [reply base/uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher/src/uk/ac/stfc/isis/ibex/e4/ui/perspectiveswitcher/views/PerspectiveSwitcherView.java]: 👍

**Pull request merged**